Source/WebCore/platform/graphics/cg/ColorCG.cpp

```diff
@@ -212,6 +212,20 @@ RetainPtr<CGColorRef> cachedCGColor(const Color& color)
     return cache.get().get(color);
 }
 
+RetainPtr<CGColorRef> cachedCGColorFor(const Color& color, CGColorSpaceRef colorSpace)
```

Comment:
Given that this function does something non-trivial, I think it needs a more clear name to indicate it has this behavior for extended range color spaces.

Reply:
Also agreed :)

I'll try think of something better than `cachedCGColorMaybeMappedIntoSDRRangeFor` (and add a settings check for this behaviour).